src/app/games/connect-six/tests/ConnectSixMinimax.spec.ts

```diff
@@ -0,0 +1,169 @@
+    });
+    it('should propose only one move at first turns', () => {
+        // Given the minimax and the ConnectSix size of board, and the initial board
+        const width: number = ConnectSixState.WIDTH;
+        const height: number = ConnectSixState.HEIGHT;
+        const state: ConnectSixState = ConnectSixState.getInitialState();
+        const node: ConnectSixNode = new ConnectSixNode(state);
+
+        // When asking it the list of move at the first turn
+        const moves: ConnectSixMove[] = minimax.getListMoves(node);
+
+        // Then it should only include the center of the board
+        const cx: number = Math.floor(width/2);
+        const cy: number = Math.floor(height/2);
+        expect(moves.length).toBe(1);
+        expect(moves[0]).toEqual(ConnectSixFirstMove.from(new Coord(cx, cy)));
+    });
+    it('should do winning move when one is possible', () => {
+        // Given a board where there is place for a victory of first player
+        const board: Table<PlayerOrNone> = [
+            [O, O, O, O, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+        ];
+        const state: ConnectSixState = new ConnectSixState(board, 2);
+        const node: ConnectSixNode = new ConnectSixNode(state);
+
+        // When asking what is the best move
+        const bestMove: ConnectSixMove = node.findBestMove(1, minimax);
+
+        // Then it should be that victory
+        expect(bestMove).toEqual(ConnectSixDrops.from(new Coord(4, 0), new Coord(5, 0)).get());
+    });
+    it('should block double-open fives at level two', () => {
+        // Given a minimax at level two
+        // And an board where current opponent could win if nothing is done (..XXXXX..)
```

Comment:
a board
"could win if nothing is done" c'est faux, si il joue pas il gagne pas ! "could win whatever is played" maybe

Reply:
ok pour "could win if current player does not block them"